<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.transaction.interceptor.RollbackRuleTests" tests="9" skipped="0" failures="0" errors="0" timestamp="2021-03-04T16:40:22" hostname="WIN-LQAMLJ4MN5U" time="0.018">
  <properties/>
  <testcase name="ctorArgMustBeAThrowableClassWithNonThrowableType()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.002"/>
  <testcase name="ancestry()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.003"/>
  <testcase name="alwaysTrueForThrowable()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="foundImmediatelyWithString()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="ctorArgExceptionStringNameVersionWithNull()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="foundEnclosedExceptionWithEnclosingException()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="ctorArgMustBeAThrowableClassWithNullThrowableType()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.003"/>
  <testcase name="foundImmediatelyWithClass()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="notFound()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
